src/components/Homepage/TrustTherapistSection.jsx

```diff
@@ -2,8 +2,8 @@ import Image from "next/image";
 import Link from "next/link";
 import { withTranslation } from "next-i18next";
 
-import PageIntro from "./PageIntro";
-import Button from "./ui/Button";
+import PageIntro from "../PageIntro";
+import Button from "../ui/Button";
```

Comment:
- It will be better to use `Absolute Import`

Reply:
replied